t/issue/1401.t

```diff
@@ -0,0 +1,86 @@
+
+use Test::More tests => 6;
+
+use strict;
+use warnings;
+use 5.010;
+use autodie qw(:all);
+use English qw($OSNAME -no_match_vars);
+use File::Temp;
+
+use Rex::Commands::Run;
+
+$::QUIET = 1;
+
+my $win = $OSNAME =~ m/^MSWin/mxsi;
+my $quote = q{"};
+my $space = q{ };
+my $empty = q{};
+
+sub command {
+  my ( $path, $exe, @parm ) = @_;
+  return qq{$path/$exe } . join $empty, @parm;
+}
+
+sub relative {
+  return qq{./$_[0]};
+}
+
+my $path    = File::Temp->newdir( 'path with spaces XXXX', DIR => q{./} );
+my $tail    = 'hello';
+my $parm    = 'Rex is wonderfull';
+my $testcmd = '/bin/echo';
+my $s       = run command relative($path), $tail, $parm;
+
+#on windows if the path not exists not apears on the output
+my $not_exists = ( $win && $s !~ /(?-x:$path)/smx ) ? $s : $path;
```

Comment:
I don't think I fully understand the logic here.

On Windows, trying to execute a non-existing command seems to return the string `The  system cannot find the path specified.`. Would it be better to test for that on Windows?

Reply:
The problem is that i don't know how force windows use a specific locale or canonical form for the messages.It also reports different messages on if the path exist but no the command or if both don't exist (in one of then it reports the command you send but I don't remember what).